x-pack/test/apm_api_integration/tests/alerts/rule_registry.ts

```diff
@@ -391,7 +391,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
               "apm.transaction_error_rate",
             ],
             "kibana.alert.status": Array [
```

Comment:
Probably not part of this PR, but shouldn't we use constants for the alert fields here instead of hardcoding them?

Reply:
This is a jest inline snapshot. Manually interpolating the constants here would render the auto-updating of the snapshot mute, wouldn't it?